api/apps/api/src/modules/access-control/scenarios-acl/entity/users-scenarios.api.entity.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Role,
+  Roles,
+} from '@marxan-api/modules/access-control/role.api.entity';
+import { Check, Entity, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+import { User } from '@marxan-api/modules/users/user.api.entity';
+import { Scenario } from '@marxan-api/modules/scenarios/scenario.api.entity';
```

Comment:
Is this the same issue as the one we talked about solving using some views in SQL?

Reply:
@rubvalave yes, exactly

Reply:
Henrique told me to save that for later and to try to speed up at least the addition of scenarios acl and later scenarios lock before tackling this issue (as I was taking a little bit of time to do it because not being that familiar with SQL to create the view and such).

Is this a critical blocking issue or can it be saved for later?

Reply:
@rubvalave the "critical" quantifier is hard to judge. The code works, the feature works - but sooner than later, it will cause issues in the codebase. If you can tackle it really soon, we can postpone it - just keep in mind returning back to pay some debt is often an issue.

Leaving decision to you and the team.

Reply:
I suspect this is the kind of issues that will show up more in the future, so the sooner you get used to doing it properly, the better. fix it now, unless it takes A LOT more time

Reply:
@rubvalave can you confirm this is solved?